src/lambda/config/parameterUtils.ts

```diff
@@ -3,6 +3,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+//
+// TODO: DEPRECATED. Remove this after removing support for the legacy
```

Comment:
Are there other files that should be marked deprecated that involve 'template.json'?

Reply:
yeah